languages/javalang/java-base/src/main/kotlin/io/vrap/codegen/languages/java/base/JavaBaseTypes.kt

```diff
@@ -15,7 +16,7 @@ import kotlin.reflect.KClass
 
 object JavaBaseTypes : LanguageBaseTypes(
 
-    objectType = fromDefaultJavaType(Object::class),
+    objectType = fromDefaultJavaType(JsonNode::class),
```

Comment:
actually this should be 
```suggestion
     objectType = fromJavaType(JsonNode::class), 
```
instad of `objectType = fromDefaultJavaType(JsonNode::class), ` since JsonNode is not native to java.